modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -58,9 +58,11 @@ class NDB_Form_User_Accounts extends NDB_Form
                 if ($this->isCreatingNewUser()) {
                     return true;
                 }
-                return in_array(
-                    $editor->getData('CenterIDs'),
-                    $user->getData('CenterIDs')
+                return array_intersect(
```

Comment:
@driusan 

This part applies when the $editor is not the superuser, and has 'user_accounts' permission but NOT 'user_accounts_multisite'; I am assuming the $editor will be at least part of all the sites of the $user they are editing to return true.